blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeGraphBuilder.java

```diff
@@ -222,86 +222,137 @@ private boolean isParentOf(FlowNode node, FlowNode child){
         }
         return false;
     }
+    
+    @Override
+    public List<FlowNodeWrapper> getPipelineNodes() {
```

Comment:
This is old legacy implementation, disabled by default and to be removed so don't waste your time on it:)